tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -127,93 +137,155 @@ def test_monitor_recovery(
-            new_md5_sum.append(pod.cal_md5sum(pod_obj, self.filename))
+            md5 = pod.cal_md5sum(pod_obj, self.filename)
+            new_md5_sum.append(md5)
+            logger.debug(f"MD5sum for pod {pod_obj.name}: {md5}")
         logger.info(f"Md5sum calculated after recovery: {new_md5_sum}")
+
+        logger.assertion(
+            f"Verify md5sum matches after recovery: original={self.md5sum}, new={new_md5_sum}"
+        )
         if collections.Counter(new_md5_sum) == collections.Counter(self.md5sum):
             logger.info(
                 f"Verified: md5sum of {self.filename} on pods matches with the original md5sum"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)